Classes/Classes/miniphonedir.cpp

```diff
@@ -0,0 +1,101 @@
+#include<iostream> 
+
+using namespace std; 
+
+  
+
+class UserData { 
+
+private: 
+
+    string name; 
+
+    string last_name; 
+
+    string email; 
+
+    string phone_number; 
+
+    /*Method*/ 
+
+public: 
+
+    void set_name(string name_temp); 
+
+    string get_name(); 
+
+    
+
+     
+
+}; 
+
+  
+
+void UserData::set_name(string name_temp) 
+
+{ 
+
+    name = name_temp; 
+
+} 
+
+  
+
+string UserData::get_name() 
+
+{ 
+
+    return name; 
+
+} 
+
+  
+
+int main(void) 
+
+{ 
+
+    int option; 
+
+    string name; 
+
+    do { 
+
+        cout << "Directorio Telefonico\n"; 
+
+        cout << "Seleccione una opcion:\n\n"; 
+
+        cout << "1. Agregar Nuevo\n\n"; 
+
+        cout << "Seleccione:"; 
+
+        cin >> option; 
+
+  
+
+        switch (option) 
+
+        { 
+
+        case 1:  
+
+            system("cls"); 
+
+            UserData User_New; 
+
+            cout << "Nombre:"; 
+
+            cin >> name; 
+
+            User_New.set_name(name); 
+
+            cout << User_New.get_name(); 
+
+            system("pause"); 
+
+        } 
+
+    } while (option != 4); 
```

Comment:
We can add a new option to display the current contacts stored using range based for loop:
void print(const std::list<UserData*>& list)
{
    for (auto const& i: list) {
        std::cout<<"Name: " << i->getName();
        std::cout<<"Phone number: " << i->getPhoneNumber();
    }
}